diff --git a/customizepkg b/customizepkg
index eac7d63..dd1bab8 100755
--- a/customizepkg
+++ b/customizepkg
@@ -120,9 +120,9 @@ modify_file()
 				elif [[ -f "${pattern}" || -f "${CONFIGDIR}/${package}.files/${pattern}" ]]; then
 					echo "=> apply patch ${pattern} using '-p${context}'"
 					if grep -q '^[[:blank:]]*prepare()' "${scriptfile}"; then
-						sed -i "/^[[:blank:]]*prepare()/{n;s%$%\npatch -Np${context} -i ${pattern}%;}" "${scriptfile}"
+						sed -i "/^[[:blank:]]*prepare()/{n;s%$%\npatch -Np${context} -i \$srcdir/${pattern}%;}" "${scriptfile}"
 					else
-						sed -i "/^[[:blank:]]*build()/{n;s%$%\npatch -Np${context} -i ${pattern}%;}" "${scriptfile}"
+						sed -i "/^[[:blank:]]*build()/{n;s%$%\npatch -Np${context} -i \$srcdir/${pattern}%;}" "${scriptfile}"
 					fi
 				else
 					echo "error: file not found '${pattern}'. Try putting it in ${CONFIGDIR}/${package}.files"
