diff --git a/PKGBUILD b/PKGBUILD
index 3f3cf3e..9bbc454 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -18,11 +18,7 @@ provides=('gqrx')
 _gitroot=git+https://github.com/csete/gqrx.git
 _gitname=gqrx
 source=("fcd_in_gnuradio.patch" "21-fcd.rules" "gqrx.png" "gqrx.desktop" "$_gitname::$_gitroot")
-md5sums=('95292c41a5d521c023a26d615a5e2478'
-         '2be3bf7cba02e90cbbb9d5f6cfdaf68b'
-         'f7032a8883c89bd80e0d0fd36f861c59'
-         '810f89195231c18f32af92522aade721'
-         'SKIP')
+
 sha256sums=('910bb64e475cfadc8f3ac51e2a3bb61f42a7a2ec55a60aa9121d213c651f2d73'
             '490fdc3e16c8d6ae38894896f622531e2bf617963165ce2430a3c132d91ba5e5'
             'abb0c2598830c9fe728d8cd0d2f187d10331f7cb607520ff3664afd67a3a279d'
